btree/btree.c

```diff
@@ -2421,6 +2473,8 @@ static void ff_fini(const struct nd *node)
 		.ot_type          = 0,
 		.ot_footer_offset = offsetof(struct ff_head, ff_foot)
```

Comment:
ot_footer_offset should also be set to 0.

Reply:
done